### DISCUSSION TITLE
With the addition of 'virial' information, the training accuracy is rather worse

### DISCUSSION THREAD

**Discussion by ZKNB** [deepmodeling/deepmd-kit#3848] [Q&A]
For the same system, adding stress information makes the accuracy worse instead, here are my test results:
trainingdata | validationdata | rcut_smth | rcut | embedding net | fitting net | decay_steps | start/limit_pref_e | start/limit_pref_f | start/limit_pref_v | sel | Energy RMSE eV | Energy RMSE/Natoms | Force  RMSE    eV/A | Virial RMSE/Natoms
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
21412 | 0 | 0.50 | 6.50 | 25 50 100 | 240 240 240 | 50000 | 200/200 | 15/15 | no | auto | 3.23E-01 | 1.05E-02 | 2.10E-01 |  
21412 | 0 | 0.50 | 6.50 | 25 50 100 | 240 240 240 | 50000 | 200/200 | 15/15 | 1/1 | auto | 4.90E-01 | 1.49E-02 | 2.64E-01 | 3.23E-01

As I understand it, adding stress should make the accuracy higher, but it turns out to be the other way around, I want to train the accuracy to an order of magnitude of 1e-3 for energy and an order of magnitude of 1e-2 for force, how do I tweak the parameters, can you give some advice?

**Comment by wanghan-iapcm** [comment]
it is highly possible that the virial labels are not consistent with energies and forces. please check the convergence of virial with respect to the energy cutoff and k-spacing used in your dft calculations.
